plugin/action/mask/mask.go

```diff
@@ -57,14 +59,15 @@ type Mask struct {
 	//> @3@4@5@6
 	//>
 	//> Regular expression used for masking
-	Re string `json:"re2" default:"" required:"true"`
+	ReStr string `json:"re2" default:"" required:"true"`
```

Comment:
Another plugins use `Re` / `Re_` naming. `Re` is a string, `Re_` is a compiled regexp. And why it's `re2` in json?